### PR TITLE
Add specifications and documents related to the Hungarian tables

### PR DESCRIPTION
**Pull request by hammera**
Hi @egli or @bertfrees,

I doed a hungarian subpage related hungarian/README.md file to [fix issue #1164](https://github.com/liblouis/liblouis/issues/1164).
I add Hungarian Braille Commission page hungarian language downloadable documents links a bulleted list, described the hungarian braille dots what worksheet what type dots examples containing.
I upload the larger grade 2 hungarian language rule set containing word document the hungarian/hungarian_large_rules.doc file.

If you have a little time, please review this change, and if I your openion not need doing any work the hungarian subpage, please commit this change to the source repository.

In the 3.22 release cicle I will doing an english language developer perspective documentation with special braille input hungarian rules related.

This hungarian subpage purpose to easyest other Liblouis maintainers life if need doing any little change into hu-hu-comp8.ctb, hu-hu-g1.ctb or hu-hu-g2.ctb tables.
The subpage I wrote the official e-mail address and the Liblouis list used e-mail address, if you would like and other table maintainers usual not describe he's e-mail address, feel free remove this information.

The second commit not need reviewing, only fixing few places into the hungarian/README.md file the bulleted lists, the first commit I doed wrong markdown format the bulleted lists.

If you think the subpage is too detailed or contains too much unnecessary information, feel free to edit the document, narrowing it down as you please. Feel free to rewrite the document, this is just a sample page.

hopefully easyest your work this change.
Have a nice day, and thank you your patience,

Attila

### CODE REVIEW

**Comment by bertfrees** [comment]
Thanks!

We will have to do some rewriting because the English is not always very understandable.

> In the 3.22 release cicle I will doing an english language developer perspective documentation with special braille input hungarian rules related.

OK good. But could I ask you to write this documentation in Hungarian please? It is mostly relevant for Hungarians anyway, and if others want to read it, they can use Google translate.

File: hungarian/README.md
Comment (bertfrees) [inline hungarian/README.md]: All this information is about the Liblouis tables, and about you, the maintainer of the tables. I think it would be better if this information would go in the tables themselves. With the "braille-specs" sub-project we try to provide general information that goes beyond the implementation.

File: hungarian/README.md
Comment (bertfrees) [inline hungarian/README.md]: I'm not sure if this is relevant information (for Liblouis)?

**Comment by hammera** [comment]
Hi Bert,

You wrote yesterday:
> We will have to do some rewriting because the English is not always very understandable.

Ok, thanks, feel free change anything in this documents, because this is a sample subpage structure only.

I wrote yesterday, and you answered following:
> In the 3.22 release cicle I will doing an english language developer perspective documentation with special braille input hungarian rules related.
OK good. But could I ask you to write this documentation in Hungarian please? It is mostly relevant for Hungarians anyway, and if others want to read it, they can use Google translate.

Clear, thanks the answer. My previous purpose to I only want doing english language developer documents because other subpage, for example Irish braille specification subpage see similar version. But you absolute right. I will doing I think two document, if you accept this:
* An easyest understandable normal Excel table, with containing only the dot combinations, and proper unicode braille simbols: this document not containing any comment rules, and easyest to use other Liblouis maintainers.
* A detailed, you requested hungarian language rule set documentation, similar with other Hungarian Commission linked subpage available rule set documents with describes what rules available in hungarian braille standards, and each rule set what the purpose. I uploaded a similar document to the hungarian/large_hungarian_document_rules.doc file with containing larger grade 2 rules reserved abbreviations, exceptions, etc. The linked Excel table only the 45 small abbreviations rule set available the 4. worksheet, and this Excel table not containing braille exception examples with grade 2 rules related.

**Comment by hammera** [comment]
@bertfrees, yesterday you wrote folloving after review the next comment:
> All this information is about the Liblouis tables, and about you, the maintainer of the tables. I think it would be better if this information would go in the tables themselves. With the "braille-specs" sub-project we try to provide general information that goes beyond the implementation.

Ok, clear answer. Please remove the wrote paragraph. In all hungarian table files since creation time described my official maintainer address, the Liblouis source tree main directory inside the AUTHORS file my both two e-mail address already described in the „TABLE AND TEST CONTRIBUTORS” section too. You are absolute right, this paragraph are unneed. Previous, when I wrote this subpage document I don't known what the general accepted rule in the braille specifications subpages, need describing Liblouis community side official table maintainer contact informations or not. Sorry the unneed paragraph. If you would like and you now have another jobs with another pull requests, I will remove this paragraph. I understand now absolute this subpage purpose is try to provide general information that goes beyond the implementation, and this subpage purpose is not a developer or technical subpage. :-):-)

The next comment after review you wrote with Marburg Medium standard related:
> I'm not sure if this is relevant information (for Liblouis)?

If other subpages not describing this general standard (or perhaps country level accepted variant), feel free remove this general paragraph. I translated previous only this paragraph to english language because in Hungarian Braille Commission subpage available this general information. But you are absolute right, Liblouis side is total irrelevant the Marburg Medium standard (dot distances). This is interesting only with printing houses, or Braille embossers manufacturers (for example [ViewPlus Technologies](https://viewplus.com/), [Index Braille](http://www.indexbraille.com), etc.). You are right, for Liblouis side real unneed this paragraph.
Note: Marburg Medium standard declaration metrics are not world wide accepted standard?

**Comment by bertfrees** [comment]
> I will doing I think two document, if you accept this:
> - An easyest understandable normal Excel table [...]
> - A detailed, you requested hungarian language rule set documentation [...]

Perfect, thanks.

> > I'm not sure if this is relevant information (for Liblouis)?
>
> If other subpages not describing this general standard (or perhaps country level accepted variant), feel free remove this general paragraph. 

OK :-)

> Marburg Medium standard declaration metrics are not world wide accepted standard?

I don't know.

**Review by egli — APPROVED** [review]

**Comment by hammera** [comment]
@egli, thank you the review and the grammar check corrections. Previous I doed lot of grammar check mistakes unfortunatelly, my english is not always perfect.
I think this document is absolute fine, I think I not need doing any change now the future hungarian Braille subpage.

If this pull request is closed, I will paste the final hungarian Braille specification subpage link the metadata tables (affected files I think tables/hu-hu-comp8.ctb, tables/hu.tbl, tables/hu-hu-g2.ctb).

I have an already opened and 3.22 release scheduled pull request, so because I working this pull request, if you or Bert send me the final link, I will simple put the new link the affected metadata tables.

I would like wish good public holiday you and Bert with june 6, and wish very good Liblouis 3.22 release in june 7. Pentecost is also a public holiday in Hungary on June 6.

I think the Liblouis 3.22 release will be a fantastic release, containing lot of bug fixes (Samuel doed bug fixes and optimizations), If I remember right happened code cleanups, perhaps before 3.22 release will be have other language table improvements. In Github repository side happened lot of changes in the CI/CD workflows, so, the 3.22 release will be again a very good step forward.
So, thank you your and Bert very hard work.

Future I very would like doing an easy YAML test file generator program for beginner table maintainers and the Liblouis community (similar the lou_genharness old utility), this program already supports hungarian language.
The generator program is big percentage ready, only the automatic international table detection is not resolved. So, for example if the maintainer input the language two letter code or the proper language code, and perhaps input the grade 1 or grade 2 level, the program need founding the proper language table (for example the hu-hu-comp8.ctb, hu-hu-g1.ctb, hu-hu-g2.ctb, etc).

All basic tests flags are already ymplemented and works perfect during YAML database generation (forward, backward, bothDirections, hyphenate test generation), but need me cleaning the code before I send review the tool (for example need doing proper exceptions handling and code cleanup).

If you and bert accept this idea, I think this utility easying the very beginner table maintainers thing to generate proper test YAML databases for you (not need doing lot of monotone work by hand with the table maintainer). We future need found a solution in Python3 to the test generator program found the proper language table.
In Python3 possible founding the right table variation, if a language have for example three Braille variations (for example english language have lot of standards, UEB Braille, not UEB Braille, grade1, grade2, and english language dialects, variations).

Liblouis Python3 bynding supports table metadata level searching or table finding?
Anybody implemented the lou_findtable, similar with C Liblouis code?
I known better the Python3 language (this is my main programming language my work place), so I am not an expert C language programmer.

Kind regards,

Attila

**Comment by bertfrees** [comment]
> Future I very would like doing an easy YAML test file generator program for beginner table maintainers and the Liblouis community (similar the lou_genharness old utility), this program already supports hungarian language.

Hi Attila. That's interesting. Can you say something more about it? How will it work? (A brief explanation is fine 😉).

> Liblouis Python3 bynding supports table metadata level searching or table finding?
> Anybody implemented the lou_findtable, similar with C Liblouis code?

There are no Python bindings for `lou_findtable` yet. You are very welcome to add it if you know how to do it. Otherwise I think maybe @sthibaul might be interested in doing it. (See https://github.com/liblouis/liblouis/issues/1202.)

File: hungarian/README.md
Comment (bertfrees) [reply hungarian/README.md]: This issue is not resolved yet. I don't think this information is relevant for Liblouis.

File: hungarian/README.md
Comment (bertfrees) [reply hungarian/README.md]: This issue is not resolved yet.

File: hungarian/README.md
Comment (bertfrees) [inline hungarian/README.md]: Could you please add some more info (author, Hungarian title, year, publisher, ISBN, etc., whatever is available) about these documents?

Also, I think it might be useful to (if allowed) put copies of the documents on our website instead of linking to them. The reason is because there is always a risk that links become dead after some time.

**Comment by hammera** [comment]
@bertfrees, with test generator program a brief explanation:
When I need do oldest time the hungarian YAML test corpus databases for Liblouis side, I not would like doing manual the testcase files after the harness migration for yaml test base (need fast do a solution when the migration happened). So I wrote a similar program.
The program accepts following basic parameters (command line, I not wrote GUI part the program for example in GTK3):
* Input file name: this source file containing the input test corpus list (for example a plain text file, with containing words, separated with \n, or \n\r characters).
* Output file: this file will be the destination file, with will be the yaml file.
* Language: for example this is now me purpose is hungarian, but if lou_findtable will be implemented, future possible not need.
* Grade level:  if I put for example 1, the program put the test file with hu.tbl the proper place. Future this situation if the table finding is resolved, the program fill the proper language depending international table column the yaml file. If I put 2, the Yaml generator use for example hu-hu-g2.ctb file.
* Table names, separated with spaces. This parameter we will be remove future if table finding is resolved. Any table names are possible input, because Liblouis Python3 binding only uses this input parameters with forward translation, backward translation, hyphenate tests, etc.
* Test direction toggle: supports the program with forward, backward, hyphenate, I think hyphenateBraille, bothDirection modes. This is the last parameter. If everithing is fine the sintax, the program creates any size YAML test database from the input file to the output yaml file, the proper test flag direction.

Very similar the solution method with old lou_genharness Python3 utility with free time created oldest time Mesar and me, when the Liblouis community used the harness unit test system for Braille testing. The old test system when the yaml migration happened is changed for the YAML test system, this is fine, the YAML test file solution is perfect and flexible solution.
So you see, independent to the test generation solution is works with hungarian language, the code implementation is not international community ready yet (I don't no how possible detecting the difference if an english language user wants ueb or not ueb braille, grade 1, grade 2, grade 3 braille, or wants en_uk or en_US dialect braille). Need me doing some code cleanups, exception handling before I open a final pull request the YAML generation related. Perhaps this is possible happening in 3.23 cicle if we found a solution.
Absolute sure very beginner table maintainers thing possible easying, when he's not known yet the YAML test sintax detailed, but would like open a pull request a new table related and want ship a test corpus database the new table (this is the minimum to you see everithing works fine the new implemented table).
Lot of time need requesting the new beginner community table maintainers to put a new proper YAML test database the new implemented tables (some time this is possible forgot doing the new table maintainer the new testcase file. Perhaps the very beginner maintainer happy to doing the new table, simple want opening the pull request, but simple not known the proper YAML test file sintax).

If not need future hand manual work doing the testcase file, you enough to say the beginner maintainer:
„Ok, your table solution looks good, this is big step forward, the harder work you doed fantastic. Please create relative big good word corpus list plain text file (one word /line), and run the lou_genyamltest utility before open the pull request.” :-):-)

**Comment by hammera** [comment]
I Agree the paragraph removal (marburg medium standards related), , if other braille specification documents not containing this information.
If I known right (I am not full sure this), Marburg medium standard is an international standard, but this standard only need knowing with Braille Embossers manufacturers and braille printing houses (for example Viewplus Technology, Index Braille embosser manufacturers, etc), Liblouis level not relevant this standard, table maintainer community persons not need this information.
Chris, what do you think? We drop this paragraph the subpage document?

If yes, feel free drop anybody this paragraph, or if you and Bert need doing other bigger review or coding works, I will doing this before put a fixup commit.

Bert suggested more acceptable and full right changes (insert not external links, add new files with available now only the hungarian webpage, etc).
So, I need add the .brf documents in the hungarian subpage and not good linkng the hungarian external link (Bert you true, this is preventing link deading, thanks the suggestion, I forgot previous this).

**Comment by hammera** [comment]
Bert, what the suggestion the maintainer part related? Only need remove the private and full work time e-mail address, and because I subscrybed the list, simple ask the maintainers ask a hungarian table maintaining or change related question me in the Liblouis List? If this is the case, I full understand what you would like, I now see the duplicated part between I think the 40. and 47. lines.


Chris now doed following part from the 27. line in hungarian/README.md:
„Official table maintainer from Liblouis community in Hungary: Attila
Hammer from [IT Foundation for the Visually Impaired -
Hungary](http://www.infoalap.hu). Attila is subscribed in Liblouis
list. If anybody have question the Hungarian braille tables related,
feel free ask a question in developer community.

Before modifying the tables please ask the table maintainer. If you
would like to do bigger changes in the Hungarian braille tables,
please do not commit the changes to the [Liblouis
repository](https://github.com/liblouis/liblouis) until all test cases
are passing, because Hungarian language is a very complex language.”

So, between if I see right the 27. line and the 39 line the paragraph part is right and fine, correct.

In the 40. line I see real following:
„If a maintenance change related have a question, please write e-mail
to the table maintainer with following addresses:

* Full official work time used e-mail address, with available the
  Hungarian braille tables: hammer.attila at infoalap dot hu
* Private e-mail address, with Attila subscribed into Liblouis mailing
  list: hammera at pickup dot hu”

Perhaps I understand what you would like. Oh my good, you not would like simple drop this part (between the 40. line and the 47. line part)? If this is the case, this is real need, because between in the document the 27. line and the 39. line is full perfect and acceptable. :-):-)
Oh my God, why didn't I think of this completely simple solution before? If I simple subscribed the Liblouis community list, community maintainers ask me any questions in the list. :-):-)
Real, this solution is full perfect, feel free do it, or I will do it if you agree.

**Comment by hammera** [comment]
Bert, the two .brf documents I will putting the subpage directory, this is not problem, because the document format is not change.
If I known right, this documents not have ISBN number, but I will looking for you.
The excel table absolute sure not have author. Github stores right binary format an excel format? So, safe I commit an excel table in Github repository?

Two external link I not would like remove if this is possible:
+* [Summary collection of decisions concerning Braille adopted by the presidency of the Hungarian National Association of the Blind and Visually Impaired](https://www.mvgyosz.hu/wp-content/uploads/2020/04/braille-%C3%ADr%C3%A1s-eln%C3%B6ks%C3%A9gi-hat%C3%A1rozatok.docx)
+* [The medicine boxes braille standards collection documents (dot sizes, line length, etc.)](https://www.mvgyosz.hu/wp-content/uploads/2020/04/braille-gy%C3%B3gyszercsomagol%C3%A1sokon.doc)
Unfortunatelly this two documents have chance random time change or update, and because I not have commit access the repository, If the presidency of the Hungarian National Association of the Blind and Visually Impaired accept a new rule change or dot combination change and update the homepage the document, I not have simultan update the repository the collection rule.
So this situation the homepage dinamical update the collection document.

So:
* The two .brf document I will put the hungarian directory if you agree this (if have author or ISBN numbers this documents, I complete the link part the proper informations after I looked this),
* Depending safe to store an excel table in Github or not, I will put the .xlsx file the hungarian directory,
* If you agree this, keep only the my last part wrote external links.

**Comment by bertfrees** [comment]
> Bert, what the suggestion the maintainer part related? Only need remove the private and full work time e-mail address, and because I subscrybed the list, simple ask the maintainers ask a hungarian table maintaining or change related question me in the Liblouis List? If this is the case, I full understand what you would like, I now see the duplicated part between I think the 40. and 47. lines.

I would like the information about the Liblouis table and the maintainer of the table to be removed from the braille-specs repo completely. You are the maintainer of the table, not the braille standard. So while this information is very good to have in the Liblouis table header, it does not belong in this repo.

I guess if people have questions about the braille standard, it could be useful to have a contact email address, but that would be an email address of the Hungarian Braille Committee.

> The two .brf document I will put the hungarian directory if you agree this (if have author or ISBN numbers this documents, I complete the link part the proper informations after I looked this),

OK. Just add any the information that is available. The idea is to identify the documents as good as possible, but if it has no ISBN, that's not a problem whatsoever.

> Depending safe to store an excel table in Github or not, I will put the .xlsx file the hungarian directory,

Excel files are fine if they add value. Usually we only publish Excel files if no better documentation is available, but if the Excel file helps to clarify what is in the other more official documentation, it's fine. You decide.

> If you agree this, keep only the my last part wrote external links.

In general, I think even if an online document gets frequent updates, it is still good to have a copy on our website. I would even argue it is preferable because this way you can make a distinction between the version of the standard that Liblouis implements (the copy of the document on our website, with mentioning of the date when it was last updated), and the latest version of the standard (a link to the official website).

If the Liblouis table is maintained very actively and always implements the latest standard, it's less important.
 
> The medicine boxes braille standards collection documents (dot sizes, line length, etc.)

Are you sure this is relevant for Liblouis?

**Comment by bertfrees** [comment]
@hammera Regarding the test generator program: if I understand correctly it automatically generates a dictionary style test file from a corpus, by running all words through Liblouis.

I can see the benefit of having such tests, especially for detecting regressions. But I think we have to be cautious not to create big blobs of test data that is not actually verified by anyone, and simply regenerated whenever a change is made to the table in order to not have to verify it. We want to encourage people to contribute carefully written tests. That means: with good coverage, yet not too extensive so that it is maintainable and readable, well documented, verified, and with `xfail` flags to mark tests that Liblouis currently fails to translate correctly. There is a risk that a tool like you suggest undermines this goal.

This is just a word of caution. Overall I think your suggestion is good.

**Comment by hammera** [comment]
Maintainer part:
Oh, I understand Bert you what would like. Unfortunatelly the commission not have public e-mail address.

So, we simple need drop from the hungarian/README.md file the document between the 27. lines and the 47. lines part. Correct I understand? If yes, this is fine and absolute correct.
Table maintainer related informations are declared the hungarian table files, you Bert are absolute right. Braille commission only determine the new rule changes (this is an organization in hungary), but developer perspective I only simple always refreshing in Liblouis hungarian tables the actual changed Braille rules. This is real independent the braille standard declaration maintenance, you absolute right. Previous i Misunderstood the subpages concept purpose.

When I created the draft release hungarian Braille specification subpage, previous I full simple missunderstand the subpages purpose. I thinked previous the language-specific Braille subpages purpose is to collect Liblouis maintainers and developers the actual language awailable standard rule documentation, and the actual language table maintainers related informations. :-):-)
Excuse for you the missunderstanding, I will correcting this part.

Medicine boxes related informations:
Bert, you are true. Perhaps this is similar with the Marburg Medium standard. For example this linked document need know with only with printing houses this document or braille embossers manufacturers, but not for Liblouis developers or Liblouis maintainers.
So, if you and Chris would like, I will remove both the Marburg Medium standard related document part and with you last showed link (the medicine boxes related document).

With test generator program related your caution I full understand and full acceptable.
This is full respectable, and you absolute true, thank you the caution warning. I understand why you say me this.

We need careful estimate lot of thinks, risks before give beginner table maintainers an automatic corpus dictionary test database generator program.
Of course the test generator program future will be not the purpose to beginner maintainers manufacture irreal big test corpus databases without documentation, verification, and maintenance.
Perhaps future when the generator code will be final specified, reviewed and committed need write a caution warning for maintainers in the list, to not manufacture irreal big test databases, because in Github we have only limited month minute time period to run workflows actions.

So, I not would like future to beginner maintainers manufacture more hundred thousands words containing test corpus databases files (without verification and lazy maintenance), my purpose only easyest the corpus test file generation related work.

More hundred thousands words corpus test database irreal increase workflow action speed running time, and irreal big increase the source repository size.
This more hundred thousand big corpus test databases need store the maintainer he's local machine, if have he a sure Braille verifyed word list.

For example, we doed my colleagues previous when begun implementing hungarian Braille tables:
Oldest time when in hungary me and my colleagues beginning implementing the hungarian braille tables, previous we split the word list and correct read manual all words to prevent wrong test corpus result. When we final read the original corpus, we generated a very large test database, but local machine only we store this list.
This is the difficulter, but right maintaining method.

You see right the implementation purpose, the generator simple do automatically the previous manual hand need doing work (lou_translate, lou_backtranstlate style works) when the test file generation happening.
For example, when I used a corpus file, I of course put previous full right verifyed braille readable words.
We need careful the repository size limit in Github, and need carefull the .github/workflows actions running time period, because I think Github provides a limited number of minutes per month to run action operations. For example in Gitlab provides 50000 minutes limited per month to run action operations, but I don't know this time period/month in Github.
Depending the test case files size and numbers, increasing the action time speed time value will be happening. So bert, very thank you the caution for me, you absolute true, we need estimate lot of thinks before we give by hand the beginner maintainers hand an automatic test generator program.
I think we talked everithing the subpage related, I go to working and fix you already suggested and conclusive accepted changes.

**Comment by hammera** [comment]
@bertfrees, following tasks are done:
* Fortunately I founded the braille alphabet books the publisher (MVGYOSZ), the creator (the braille commission with proper I wrote the hungarian/readme.md file), the release year in 2018, and the ISBN numbers with you request for me. I added these informations before the documents links.
* I putted the .brf documents in the hungarian directory, and remove the external links. Hopefully will be good the accented filenames links. If not, need rename the .brf files.
* I putted the hungarian dot combination excel table the hungarian directory. I hopefully good the accented filename. If not, need rename the .xls file.
* I dropped from hungarian/README.md file the maintainer related part from the 27 line to 47 line.
* Temporary I keep the marburg medium standard, and two remaining links. The medicine boxes related standard will be not need I think, because this is the printing houses standard, this is the similar case I think the marburg medium standard too.
* Because the rule collection externeal link document not changed since 2019 (I will looking this before june 1 again), I will put the last external link awailable document the hungarian directory if we known good the accented filename links the homepage or not. Because Liblouis Braille Specs page is not a Jekyll site, I not have possibility to try an offline test before publishing.

Hopefully I not forgot anything. If anybody I forgotted, please tell me and I doing tomorrov morning.

**Comment by hammera** [comment]
@bertfrees, I hopefully don't forget anything for you the requested changes implementing.
So, now I putted all in the hungarian webpage online awailable documents the hungarian subdirectory.
Because the Braille commission determined rule collection standard are not changed since 2019 march 14, I putted this document into hungarian directory too.
I keep the online link, if future happening any update the online version, easy possible verify the difference. Not often happening a change the rule book document, I always checking this document before Liblouis end of development cicle.

Because we talked previous, I agreed your request, and I removed medicine boxes related informations and marburg medium standard related paragraph.

So, I think now the hungarian subpage is fine (hopefully). Perhaps the 3.23 milestone we paste the proper link the hungarian main tables or the proper metadata files.

Thanks the cooperation, and wish good rest for you in monday,

Attila

File: hungarian/README.md
Comment (bertfrees) [inline hungarian/README.md]: Information about Liblouis tables shouldn't be here. Liblouis tables link to this page, the other way is not necessary.

File: hungarian/README.md
Comment (bertfrees) [inline hungarian/README.md]: I like to put the original (Hungarian) title in the link text. The English translation of the title can be added below as clarification.

**Review by bertfrees — CHANGES_REQUESTED** [review]
I'm sorry, I had written these two comments but had not sent them, they were still pending.

**Comment by hammera** [comment]
@bertfrees , please look the doed changes.
Hopefully everithing is looks noow good. If I anybody moved wrong the links related, or wrote wrong english language the documents description part with in to the clarification part, feel free edit the document.

Now I put the link texts the hungarian description text with available the hungarian commission subpage, and put english descriptions the clarification part.

Attila

**Comment by bertfrees** [comment]
Thanks @hammera

**Comment by hammera** [comment]
Hi boys,

We forgotted this PR. :-):-)
Have any remaining required changes, or have possibility to merge this PR to the braille-specs subpage and update in Liblouis the table metadata part the hungarian tables with proper link after this change is merged?
I checked the documents of the decisions made by the presidency on the braille subpage of the Hungarian Association of the Blind, there have been no changes since October 2019.
So, the Hungarian sub-site is fine in terms of the correctness of the stored documents and links.
If you have a little time, please approve this pull request at some point so that we can update the metadata parts together in the necessary places of the Hungarian table files, at the latest after the release of Liblouis version 3.31.

Attila

**Comment by hammera** [comment]
Bert, thanks the merge.
What files need update with new hungarian braille-specs link related?
I remember  right only with liblouis/tables/hu-hu-comp8.ctb, liblouis/tables/hu.tbl and liblouis/tables/hu-hu-g2.ctb files are affected the required update related?
Have a standard type text with we using in english the tables metadata part when a metadata subpage are created to replace the now used metadata part text in table files (when missing still the required metadata documentation subpage information)?
I don't would like doing for you and Chris with extra work my english language correction when I updating metadata field with affected hungarian table files, but I welcome creating a PR with evening with required table files metadata update related. :-):-)
If you would like doing an update with required hungarian table files (because perhaps this is faster method), feel free do it and paste the proper standard text.

So, thanks the good cooperation and the merge,

Attila

**Comment by bertfrees** [comment]
No, there is no standard text. It's different in every table. Just go ahead and propose something. If a correction is needed to the language that's not a problem.

**Pull request merged**
